frame/ibc/src/impls.rs

```diff
@@ -683,9 +673,14 @@ where
 			ctx.channel_counter().map_err(|_| IbcHandlerError::ChannelInitError)?;
 		let channel_id = ChannelId::new(channel_counter);
 		// Signer does not matter in this case
-		let value = MsgChannelOpenInit { port_id, channel: channel_end, signer: Signer::new("") }
-			.encode_vec()
-			.map_err(|_| IbcHandlerError::ChannelInitError)?;
+		let value = MsgChannelOpenInit {
+			port_id,
+			channel: channel_end,
+			signer: Signer::from_str("pallet_ibc")
```

Comment:
Can we make the `"pallet_ibc"` a constant to avoid typos etc. I'm seeing `"relayer"` used in the benchmarks and wondering what the difference is.